src/process/enumeration.rs

```diff
@@ -0,0 +1,43 @@
+use rayon::iter::{IntoParallelRefIterator, ParallelIterator};
```

Comment:
Rayon documentation suggests you use rayon::prelude::* instead of `use`ing everything manually